src/Tizen.Security.SecureRepository/Interop/Interop.CkmcErrors.cs

```diff
@@ -15,23 +15,46 @@
  */
 
 using System;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
```

Comment:
Why do you need these?

Reply:
I briefly created a principle of exclusion helper for the methods which did not return and throw InvalidOperationException. It seemed overkill at the moment though as we only needed this in one place. Check [here](https://gist.github.com/Mrnikbobjeff/9f5ec4110f64aa7d02fbefcaaacc2285) for my im´plementation. This is a pattern very rarely used in oop, so I thought it was too weird to put there

Reply:
Ok, thanks